src/components/timeline/TrackMemoryGraph.js

```diff
@@ -391,14 +391,16 @@ class TrackMemoryGraphImpl extends React.PureComponent<Props, State> {
     return (
       <Tooltip mouseX={mouseX} mouseY={mouseY}>
         <div className="timelineTrackMemoryTooltip">
-          <div className="timelineTrackMemoryTooltipLine">
-            <span className="timelineTrackMemoryTooltipNumber">
-              {formatNumber(operations, 2, 0)}
-            </span>
-            <Localized id="TrackMemoryGraph--operations-since-the-previous-sample">
-              operations since the previous sample
-            </Localized>
-          </div>
+          {operations > 0 ? (
```

Comment:
can we have `>= 0`? I think it makes sense to show it if it's 0.

Reply:
then I would have to use `-1` as the number of operations, but that should fine in most cases

Reply:
This is what I see in the deploy preview, that's why I suggested this. Happy to discuss this with you more to find the right solution.